.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Motivation and Context
+
```

Comment:
```suggestion
This PR resolves <#ISSUE_NUMBER>.
```

Reply:
Usually this would go under the "Related Issue", so we'd have something like:
```markdown
## Related Issue

Closes #123
```

Reply:
Github requires special syntax for this to work:https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Reply:
otherwise it links but wont autoclose

Reply:
I'm aware :) I assume people know that and will write one of the few accepted keywords before the issue number. We can always add that by default "Close #123" but the PR might not be to close an issue.